versioning/versions/versions.go

```diff
@@ -25,14 +25,18 @@ type Package struct {
 }
 
 type Version struct {
-	Dir            string
-	TemplateSubDir string `yaml:"templateSubDir"`
-	Repo           string
-	Tags           []string
-	From           string
-	TemplateArgs   map[string]string `yaml:"templateArgs"`
-	Packages       map[string]Package
-	ExcludeTests   []string `yaml:"excludeTests"`
+	Dir                  string
+	TemplateSubDir       string `yaml:"templateSubDir"`
+	Repo                 string
+	Tags                 []string
+	From                 string
+	TemplateArgs         map[string]string `yaml:"templateArgs"`
+	Packages             map[string]Package
+	ExcludeTests         []string `yaml:"excludeTests"`
+	Builder              string
```

Comment:
Optional: I think it's better to move these builder related fields into a nested struct instead.

Reply:
I think it will be better to keep it as is. For each version it can be one or two additional fields used. It will be look not good for create nested only one filed or two.